node.go

```diff
@@ -278,8 +299,8 @@ func (n *ProtoNode) Loggable() map[string]interface{} {
 // UnmarshalJSON reads the node fields from a JSON-encoded byte slice.
 func (n *ProtoNode) UnmarshalJSON(b []byte) error {
 	s := struct {
-		Data  []byte       `json:"data"`
-		Links []*ipld.Link `json:"links"`
+		Data  []byte         `json:"data"`
+		Links []*format.Link `json:"links"`
 	}{}
```

Comment:
@warpfork @rvagg @willscott does this need to be changed to `Data` and `Links`? Perhaps relatedly will this code be used anywhere anymore?